### ISSUE TITLE
SvelteKitError: Not found: /api/camera_proxy_stream/camera.

### ISSUE THREAD

**Issue by joeblack2k** [matt8707/ha-fusion#229]
I'm encountering an issue when trying to view my cameras outside of Home Assistant. I've set up Home Assistant to use port 8199 instead of the default port 8099 because port 8099 is already in use by another service called zigbee2mqtt.

When I access Home Assistant using the address "homeassistant.local:8199," it works as expected. However, I encounter a problem when I try to access the cameras directly using this address, and I see the following error message:

```
Listening on port 8099
Environment ADDON true
SvelteKitError: Not found: /api/camera_proxy_stream/camera.buienradar
    at resolve2 (file:///rootfs/build/server/index.js:4027:18)
    at resolve (file:///rootfs/build/server/index.js:3859:34)
    at Object.handle (file:///rootfs/build/server/chunks/hooks.server-hoE62lsD.js:2:26)
    at respond (file:///rootfs/build/server/index.js:3857:43)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  status: 404,
  text: 'Not Found'
}
```

Interestingly, the cameras work fine when I access them from the sidebar within Home Assistant.

In simpler terms, I'm having trouble accessing my cameras from outside Home Assistant on a different port (8199), and I'm getting a "Not Found" error when I try to do so. However, they work fine when accessed through Home Assistant's sidebar.

**Comment by abeksis** [comment]
Hey,
There are already issues that will be opened in the following link [https://github.com/matt8707/ha-fusion/issues/223](url)

I'm sure matt8707 will look into the matter further

**Comment by Mareckil** [comment]
Hi same issue here
